src/AppInstallerRepositoryCore/Microsoft/Schema/Portable_1_0/PortableTable.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PortableTable.h"
+#include "SQLiteStatementBuilder.h"
+#include "Microsoft/Schema/IPortableIndex.h"
+
+namespace AppInstaller::Repository::Microsoft::Schema::Portable_V1_0
+{
+    using namespace std::string_view_literals;
+    static constexpr std::string_view s_PortableTable_Table_Name = "portable"sv;
+    static constexpr std::string_view s_PortableTable_Index_Separator = "_"sv;
+    static constexpr std::string_view s_PortableTable_Index_Suffix = "_index"sv;
+
+    static constexpr std::string_view s_PortableTable_FilePath_Column = "filePath"sv;

```

Comment:
>"filePath" [](http://example.com/codeflow?start=72&length=10)

nit: all lower case and _ to separate if needed
please update across this file too